### PR TITLE
move some metadata dbs and types

### PR DESCRIPTION
**Pull request by lwsanty**
related to [[DEV-2459]]

[DEV-2459]: https://athenianco.atlassian.net/browse/DEV-2459

### CODE REVIEW

File: github/types/ghtypes.go [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+	"strconv"
+
+	"github.com/athenianco/cloud-common/report"
+)
+
+var ErrNoAppMeta = errors.New("no application metadata")
+
+// AthenianAccID is a Athenian Github Account ID.
+// It combines a Github AppID with a Github InstallID and can uniquely identify an installation.
+type AthenianAccID int64
```
Comment (dennwc) [inline github/types/ghtypes.go:15]: Let's take this chance to rename it to `AccID`.

File: go.mod [hunk 2]
```diff
@@ -7,11 +7,11 @@ require (
 	cloud.google.com/go/pubsub v1.6.1
 	cloud.google.com/go/storage v1.10.0
 	github.com/Nvveen/Gotty v0.0.0-20120604004816-cd527374f1e5 // indirect
-	github.com/cenkalti/backoff v2.2.1+incompatible // indirect
```
Comment (dennwc) [inline go.mod:10]: Do you know why it's gone?

File: go.mod [hunk 3]
```diff
@@ -21,5 +21,6 @@ require (
 	github.com/rs/zerolog v1.19.0
 	github.com/stretchr/testify v1.6.1
 	google.golang.org/genproto v0.0.0-20200806141610-86f49bd18e98
+	gopkg.in/yaml.v2 v2.2.7
```
Comment (dennwc) [inline go.mod:24]: Do you know why we depend on it now? I don't see anything YAML-related.

File: github/types/ghtypes.go (hunk 1 above)
Comment (lwsanty) [reply github/types/ghtypes.go]: Done.

File: go.mod (hunk 2 above)
Comment (lwsanty) [reply go.mod:10]: seems like previously we forgot to `tidy` stuff

File: go.mod (hunk 3 above)
Comment (lwsanty) [reply go.mod:24]: https://github.com/athenianco/cloud-common/pull/7#discussion_r648340445

Comment (dennwc) [reply go.mod:24]: Still, do we use YAML there?

Comment (lwsanty) [reply go.mod]: found the root cause, fixed

**Review by dennwc — APPROVED** [review]

**Pull request merged**